chapters/09-package-website.Rmd

```diff
@@ -5,6 +5,77 @@
 - Getting all resources and material organized
 - Defining lifecycle of your package and individual functions (with lifecycle)
 
+## Learning Objectives
+
+* Describe the functionality `pkgdown` provides to deploy a website for package documentation
+* Customize the website with pre-existing themes
+* Organize the website with vignettes, tutorials and custom pages
+* Explain a functions' lifecycle process and the tools to handle it in R
+
+## Outline
+
+* `pkgdown`
+
+  * Text: What is `pkgdown`?
```

Comment:
This might be a good place for a little exercise that gets students to visit a webpage for a familiar package, and have them brainstorm the advantages over R help pages.

Reply:
Added an exercise to go to the ggplot2 page and identify 3 benefits of having such a page